user-contracts-mon/internal/parser.go

```diff
@@ -0,0 +1,55 @@
+package monitor
+
+import (
+	"errors"
+	"io"
+	"os"
+	"strconv"
+	"strings"
+
+	env "github.com/hashicorp/go-envparse"
+)
+
+var invalidCfgError = errors.New("Invalid or Missing Fields in configration file")
+
+func readFile(envPath string) (io.Reader, error) {
```

Comment:
what is the point of this function? also, *os.File implements the reader interface can be used wherever you needed to use that Reader.

Reply:
This function makes it easy to test the parser without the need to create a new env file for every test case